src/pushsource/_impl/model/ami.py

```diff
@@ -99,9 +99,42 @@ class AmiPushItem(VMIPushItem):
     )
     """Billing codes associated with this image."""
 
+    release_notes = attr.ib(type=str, default=None, validator=optional_str)
+    """"Notes regarding changes in the newest version. URLs are supported.
+    
+    SPECIFICATIONS:
+    * Open text field
+    * 1 character MIN
+    * 30000 characters MAX
+    """
+
+    usage_instructions = attr.ib(type=str, default=None, validator=optional_str)
+    """Step by step instructions for the end-user to launch, configure and access the product. 
+    Write with a less-technical customer in mind. Can include URL to more comprehensive steps.
```

Comment:
Please keep a blank line after the first sentence, like this:

```suggestion
    """Step by step instructions for the end-user to launch, configure and access the product. 

    Write with a less-technical customer in mind. Can include URL to more comprehensive steps.
```

The first sentence serves as a summary, then more details can appear in later paragraphs.

Reply:
ok got it.